nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestColdStorage.java

```diff
@@ -88,8 +95,11 @@
     @Inject
     protected DownloadService downloadService;
 
+    @Inject
+    protected CoreFeature coreFeature;
+
```

Comment:
can you remove this injection and replace the open of the core session with: 
` 
try (CloseableCoreSession jamesSession = CoreInstance.openCoreSession(doc.getRepositoryName(), "john")) {
....
}
`
complete example [here](https://github.com/nuxeo/nuxeo/blob/master/nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/impl/TestTreeCommentManager.java#L240)